### ISSUE TITLE
Disable react/require-default-props

### ISSUE THREAD

**Issue by madsgodvinjensen** [trustpilot/node-eslint-config#34]
With es6 we can define default values in the destructured props argument. I vote we disable the require-default-props and use es6 default values instead. The issue with that, though, is that I don't think it's lintable.

Here's an example case that is an error, but shouldn't be IMO:

```javascript
const Component = ({ name = '' }) => {
  return (<span>Hello {name}</span>);
};

Component.propTypes = {
  name: PropTypes.string
};
```

Error: `propType "name" is not required, but has no corresponding defaultProp declaration`

**Comment by TheLaz** [comment]
👍

**Comment by madsgodvinjensen** [comment]
Fixed in release [v3.2.1](https://github.com/trustpilot/node-eslint-config/releases/tag/v3.2.1)
